tests/memories_test.py

```diff
@@ -1107,18 +1097,13 @@ def f(inp1):
     self.assertDeleted(x)
 
 
+@jtu.with_config(jax_enable_memories=True)
```

Comment:
I think you missed this one? https://github.com/google/jax/blob/main/tests/memories_test.py#L216

Reply:
Done, thanks!